### ISSUE TITLE
[Question] Loading trainer configs for resume training of ASR models

### ISSUE THREAD

**Issue by ahkarami** [NVIDIA/NeMo#3864]
Hi,
I know that when one want to fine-tune an ASR model can easily load pre-trained models' wights by using [this link hints and commands](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/asr/configs.html#fine-tuning-configurations). Now, my questions is that, how I can load the trainer configs for **resume training**? Assume, I had a long training and for some reasons, the running server shut down before the training procedure has completed. So, I want to load the last checkpoint as well as the trainer last configs (_i.e._, last learning rate value, optimizer state, etc.).
Can you please guide me how I can do this (loading the last state of trainer configs)?
Best

**Comment by titu1994** [comment]
During training, Nemo will save two different types of checkpoints - .ckpt and .Nemo (if certain exp manager flags are set). 

You will need .ckpt which is the pytorch lightning checkpoint - holding model weights as well as the optimized and scheduler. .Nemo file does not preserve any of this because it is not meant to resume training.
